back-end/insertDB.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*
+
+# input : list organized by python dictionary
+# output : print (insert success or fail)
+
+import test_list
+import pymysql
+
+attributes = ['name', 'address', 'phone_number', 'latitude', 'longitude', 'rating', 'monday_open_hours', 'tuesday_open_hours', 'wednesday_open_hours', 'thursday_open_hours', 'friday_open_hours', 'saturday_open_hours', 'sunday_open_hours', 'photourl', 'take_out_available', 'parking_available', 'bike_parking_available', 'good_for_groups', 'ambience', 'wi_fi_available']
+
+record_values = ['name_value', 'address_value', 'phone_number_value', 'latitude_value', 'longitude_value', 'rating_value', 'monday_open_hours_value', 'tuesday_open_hours_value', 'wednesday_open_hours_value', 'thursday_open_hours_value', 'friday_open_hours_value', 'saturday_open_hours_value',
```

Comment:
lines are too long. Can you break these down? https://google.github.io/styleguide/pyguide.html#Line_length

Reply:
I can use parenthesis or back slash. I changed less than 80 characters.

Reply:
I changed it !